voidcrew/turfs/closed/wall/whitesands.dm

```diff
@@ -0,0 +1,94 @@
+
+/turf/closed/mineral/iron/whitesands
```

Comment:
these are never used anywhere

I also don't think we want to use them, they're from a whitesands biome so I would think they should stay on whitesands

This goes for the whitesands cave generator/biome/whatever